src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const list = [...document.querySelectorAll('li')];
+
+function sortList(arr) {
+  const sorted = arr.sort((a, b) =>
+    converter(b) - converter(a));
+
+  ul.append(...sorted);
+
+  return sorted;
+}
+
+function converter(item) {
+  const salary = +item.dataset.salary.slice(1).split(',').join('');
+
+  return salary;
+}
+
+function getEmployees(arr) {
+  const listOfEmployees = arr.map(el => ({
+    name: el.textContent,
+    position: el.dataset.position,
+    salary: el.dataset.salary,
+    age: el.dataset.age,
+  }));
+
+  return listOfEmployees;
```

Comment:
No need to create `listOfEmployees` variable. Destructuring make it more shorter
```suggestion
  return arr.map(({innerText, dataset: {position, salary, age}}) => {
    return {
      name,
      position,
      salary,
      age,
    };
  })
```